.github/workflows/cicd.yml

```diff
@@ -116,7 +116,7 @@ jobs:
       run: |
         sha256sum ${{ needs.crate-metadata.outputs.name }}-${{ matrix.job.target }}.tar.gz > ${{ needs.crate-metadata.outputs.name }}-${{ matrix.job.target }}.tar.gz.sha256
     - name: Upload release binary
-      uses: softprops/action-gh-release@9d7c94cfd0a1f3ed45544c887983e9fa900f0564 # v0.1.15
+      uses: softprops/action-gh-release@69320dbe05506a9a39fc8ae11030b214ec2d1f87 # v0.1.15
```

Comment:
Need some investigations

Reply:
[v2.0.5](https://github.com/softprops/action-gh-release/tree/v2.0.5) refers  `69320dbe05506a9a39fc8ae11030b214ec2d1f87` but the code comment is not updated.